### PR TITLE
Update README deprecated "webpackMode" in favor of "fetch"

### PR DESCRIPTION
**Pull request by kl3sk**
After @weaverryan https://github.com/symfony/stimulus-bridge/pull/15 PR

Encore complains:
`The "webpackMode" config key is deprecated in controllers.json. Use "fetch" instead, set to either "eager" or "lazy".`

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| Tickets       | Fix #... 
| License       | MIT

### CODE REVIEW

**Review by weaverryan — APPROVED** [review]

File: src/Dropzone/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ the `@symfony/ux-dropzone/src/style.css` autoimport to `false`:
         "@symfony/ux-dropzone": {
             "dropzone": {
                 "enabled": true,
-                "webpackMode": "eager",
+                "fetch": "eager",
```
Comment (weaverryan) [inline src/Dropzone/README.md:64]: For clarification to others, the `package.json` files themselves in this repo still have `"webpackMode": "eager"`. However, that is just for backwards-compat with stimulus-bridge 1. `symfony/flex` 1.12.0 (thanks to https://github.com/symfony/flex/pull/735) will read the `webpackMode` but actually put `fetch` into your `controllers.json` file.

tl;dr - this fix is correct

Comment (kl3sk) [reply src/Dropzone/README.md:64]: But the Readme stay confusing because don't tell anything about this.

Maybe a note could be added at least to clarify this.

**Review by dunglas — APPROVED** [review]

**Comment by weaverryan** [comment]
Thank you @kl3sk!
